rebar.config

```diff
@@ -111,8 +111,7 @@
                             {include_erts, true}]}]}
            ]}.
 
-{dialyzer, [ {warnings, [unknown, underspecs, no_return]}
```

Comment:
Because then it gives errors like "Oh you say you return `string()` but here I found you a string which is never returned, so the correct typing is actually `"asdas" | "dupa" | "blah" | "I" | "hate" | "dialyzer" | "lol" | ...`